<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormularioCarreras</class>
 <widget class="QDialog" name="FormularioCarreras">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>486</width>
    <height>452</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>486</width>
    <height>452</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>486</width>
    <height>452</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Formulario Carrera</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QFrame" name="frame">
     <property name="styleSheet">
      <string notr="true">#frame{background-color: #f1f1f1;
border-top-left-radius: 10px;
border-top-right-radius: 10px; 
border-bottom-right-radius: 10px;
border-bottom-left-radius: 10px;}
</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QFrame" name="frame_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>489</width>
        <height>50</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>489</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>489</width>
        <height>50</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">#frame_2{background: #3A4F50 ;border-top-left-radius: 10px;
border-top-right-radius: 10px;}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QToolButton" name="btn_close">
       <property name="geometry">
        <rect>
         <x>454</x>
         <y>6</y>
         <width>25</width>
         <height>25</height>
        </rect>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="toolTip">
        <string>Cerrar</string>
       </property>
       <property name="styleSheet">
        <string notr="true">#btn_close{
font-family: Roboto;
font-weight: normal;
font-size: 14px;
background-color: #d5d5d8;
color: #6B7280;
border-radius:11px
}



#btn_close:hover{background:#ce4d3a ;
border-radius:11px; }

</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../image/source.qrc">
         <normaloff>:/menu/x.png</normaloff>:/menu/x.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>13</width>
         <height>13</height>
        </size>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="lbl_titulo_ventana">
       <property name="geometry">
        <rect>
         <x>19</x>
         <y>15</y>
         <width>401</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{font-family: Roboto;
font-style: normal;
font-weight: normal;
font-size: 18px;
line-height: 23px;
letter-spacing: 0.02em;
color: #FFFFFF;}</string>
       </property>
       <property name="text">
        <string>Formulario de registro de carrera</string>
       </property>
      </widget>
     </widget>
     <widget class="QScrollArea" name="scrollArea">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>50</y>
        <width>491</width>
        <height>341</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
QLabel{
font-family: Roboto;
font-style: normal;
font-weight: 450;
font-size: 14px;
letter-spacing: 0.02em;
color: #6B7280;}


QLineEdit{
font-family: Roboto;
font-style: normal;
font-weight: normal;
font-size: 14px;
letter-spacing: 0.02em;
background: #F3F4F6;
border: 2px solid #F3F4F6;
border-radius: 8px;
color:#9CA3AF;
padding:10px}

QLineEdit::focus{
background: #FFFFFF;
border: 2px solid #344647;
border-radius: 8px;
}

/* PLANTEXT*/
QPlainTextEdit{
font-family: Roboto;
font-style: normal;
font-weight: normal;
font-size: 14px;
letter-spacing: 0.02em;
background: #F3F4F6;
border: 2px solid #F3F4F6;
border-radius: 8px;
color:#9CA3AF;
padding:10px}

QPlainTextEdit::focus{
background: #FFFFFF;
border: 2px solid #3a4f50;
border-radius: 8px;
}



/* QLabel para Mostrar Error */
QLabel#errorNombre{
    font-family: Roboto;
    font-style: normal;
    font-weight: bold;
    font-size: 14px;
    color: #D32F2F; 
}

/* COMBOBOX */

QComboBox{
font-family: Roboto;
font-style: normal;
font-weight: normal;
font-size: 14px;
letter-spacing: 0.02em;
background: #F3F4F6;
color:#9CA3AF;
border-radius:8px;
padding:10px;
combobox-popup: 0;
border: 2px solid #F3F4F6;

}


QComboBox::drop-down
{
    width: 20px;
    border-left-width: 1px;
	padding: 10px 10px 10px 20px;
 }


QComboBox QAbstractItemView {
	color:#9CA3AF;
	font-size:14px;
	background-color: #F3F4F6;
	selection-background-color:#344647;
	selection-color:#ffffff;
	outline: 0px;
   border: 1px solid #344647;
 border-radius:8px;
padding:10px
}



QComboBox::down-arrow
{
	image: url(:/menu/contraerabajo.png);
     width: 11px;
     height: 11px;
}

QComboBox::focus{
background: #FFFFFF;
border: 2px solid #344647;
border-radius: 8px;
}


</string>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scroll_registro_estudiante">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>489</width>
         <height>339</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>250</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">#scroll_registro_estudiante{
	background-color: rgb(255, 255, 255);
}</string>
       </property>
       <widget class="QLineEdit" name="line_nombre">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>45</y>
          <width>441</width>
          <height>40</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="maxLength">
         <number>30</number>
        </property>
        <property name="frame">
         <bool>true</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
        <property name="placeholderText">
         <string>Ingrese  nombres</string>
        </property>
       </widget>
       <widget class="QLabel" name="lbl11">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>171</width>
          <height>21</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Nombre*</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
        </property>
       </widget>
       <widget class="QPlainTextEdit" name="plain_descripcion">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>139</y>
          <width>441</width>
          <height>91</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
       </widget>
       <widget class="QLabel" name="lbl11_2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>114</y>
          <width>171</width>
          <height>21</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Descripci√≥n ( Opcional )</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
        </property>
       </widget>
       <widget class="QLabel" name="errorNombre">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>85</y>
          <width>441</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="lbl11_3">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>260</y>
          <width>171</width>
          <height>21</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Estado</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
        </property>
       </widget>
       <widget class="QComboBox" name="cbo_estado">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>282</y>
          <width>221</width>
          <height>40</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>Activo</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Inactivo</string>
         </property>
        </item>
       </widget>
      </widget>
     </widget>
     <widget class="QFrame" name="frame_pie_pagina">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>390</y>
        <width>491</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#frame_pie_pagina{background-color: #f1f1f1; 
border-bottom-right-radius: 10px;
border-bottom-left-radius: 10px;}

/*  BOTON */

QPushButton {
    font-family: Roboto;
    font-style: normal;
    font-weight: normal;
    font-size: 15px;
    letter-spacing: 0.02em;
    color: white;
    padding: 8px 16px;
    background: #3f5758;
    border: none; 
    border-radius: 8px; 
}

QPushButton:hover {
    background: #4a6b6c; 
    border: none;
    border-radius: 8px;
}


QPushButton:pressed {
    background: #2e4546;
    border: none;
    border-radius: 8px;
}


/* Bot√≥n Cancelar */
QPushButton#cancelButton {
    font-family: Roboto;
    font-style: normal;
    font-weight: normal;
    font-size: 15px;
    letter-spacing: 0.02em;
    color: #3f5758;
    padding: 8px 16px;
    background: transparent;
    border: 2px solid #3f5758; 
    border-radius: 8px; 
}

QPushButton#cancelButton:hover {
    background: rgba(63, 87, 88, 0.1); /* Fondo ligeramente oscuro en hover */
    color: #2e4546; 
    border: 2px solid #2e4546;
}

QPushButton#cancelButton:pressed {
    background: rgba(63, 87, 88, 0.2); /* Fondo m√°s oscuro en estado presionado */
    color: #1c2d2e; 
    border: 2px solid #1c2d2e;
}

</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="btn_agregar_carrera">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>10</y>
         <width>180</width>
         <height>40</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>180</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>110</width>
         <height>40</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Guardar carrera</string>
       </property>
       <property name="icon">
        <iconset resource="../image/source.qrc">
         <normaloff>:/menu/botonadd.png</normaloff>:/menu/botonadd.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>15</width>
         <height>21</height>
        </size>
       </property>
      </widget>
      <widget class="QPushButton" name="cancelButton">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>10</y>
         <width>120</width>
         <height>40</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>40</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Cancelar</string>
       </property>
       <property name="icon">
        <iconset resource="../image/source.qrc">
         <normaloff>:/menu/no.png</normaloff>:/menu/no.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>15</width>
         <height>21</height>
        </size>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../image/source.qrc"/>
 </resources>
 <connections/>
</ui>
